src/script.c

```diff
@@ -241,7 +241,7 @@ void ScriptContext2_RunNewScript(const u8 *ptr)
     while (RunScriptCommand(&sScriptContext2) == TRUE);
 }
 
-u8 *mapheader_get_tagged_pointer(u8 tag)
+u8 *MapHeaderGetEventTable(u8 tag)
```

Comment:
The language for this system should be changed to "map scripts", instead of "map header event table"